src/binding.cpp

```diff
@@ -261,14 +262,13 @@ NAN_METHOD(render_sync) {
 
   sass_free_context_wrapper(ctx_w);
 
-  NanReturnValue(NanNew<Boolean>(result == 0));
+  info.GetReturnValue().Set(Nan::New<v8::Boolean>(result == 0));
```

Comment:
Not strings, apart from the empty string. https://github.com/nodejs/nan/blob/master/doc/methods.md#api_nan_return_value